src/main/java/com/spotify/github/v3/clients/GithubAppClient.java

```diff
@@ -37,10 +38,11 @@ public class GithubAppClient {
   private static final String GET_INSTALLATIONS_URL = "/app/installations?per_page=100";
   private static final String GET_INSTALLATION_REPO_URL = "/repos/%s/%s/installation";
   private static final String LIST_ACCESSIBLE_REPOS_URL = "/installation/repositories";
+  private static final String GET_INSTALLATION_ORG_URL = "/orgs/%s/installation";
 
   private final GitHubClient github;
   private final String owner;
```

Comment:
Can we add a comment here that owner could refer to org?

Reply:
yep nice catch!